docarray/array/abstract_array.py

```diff
@@ -16,35 +26,39 @@
 class AnyDocumentArray(Sequence[BaseDocument], Generic[T_doc], AbstractType):
     document_type: Type[BaseDocument]
     tensor_type: Type['AbstractTensor'] = NdArray
+    __typed_da__: Dict[Type[BaseDocument], Type] = {}
```

Comment:
```suggestion
    __typed_da__: Dict[Type[BaseDocument], Type] = dict()
```

Reply:
Is there an error case with using `{}`? I thought it was faster by a few milliseconds?

Reply:
![Screenshot from 2023-01-25 11-52-36](https://user-images.githubusercontent.com/55492238/214544999-e4684a4f-e8ec-4cbe-becc-d5dad9b4a608.png)

it is not safe to use {} or [] inside function argument default value

Reply:
But its a static variable... its not a default arg